addons/survey/static/src/scss/survey_templates.scss

```diff
@@ -10,3 +10,20 @@
 .js_question-wrapper {
     padding-bottom: 25px;
 }
+
+.o_survey_breadcrumb_container {
+    min-height: 3rem;
+    background-color: #e9ecef;
```

Comment:
Isn't it possible to replace it by a gray-something (gray-200 maybe) class ?

Reply:
Well I was convinced gray-200 existed too but I can't find anything about it anymore.
So... heeeeeeee not sure